scripts/rmOldKibanaIndices.py

```diff
@@ -0,0 +1,29 @@
+import requests
```

Comment:
Can we change the name of this script to removeOldKibanaIndices.py

Reply:
Sure